manifests/init.pp

```diff
@@ -346,6 +356,10 @@
   Optional[Stdlib::Absolutepath]  $ntsdumpdir                      = undef,
   Optional[String]  $ntsntpserver                                  = undef,
   Optional[Integer[0]] $ntsrotate                                  = undef,
+  Optional[Stdlib::Absolutepath] $ntstrustedcerts                  = undef,
+  Optional[Integer[0]] $nocerttimecheck                            = undef,
+  Optional[Boolean] $nosystemcert                                  = undef,
+  Optional[Enum['require','prefer','mix','ignore']] $authselectmode = undef,
```

Comment:
do I need to fix this formatting?

Reply:
IMO no, realigning just causes extra diffs and makes `git blame` less useful.